service-front/app/src/Common/src/Entity/CombinedLpa.php

```diff
@@ -22,7 +24,7 @@ public function __construct(
         public readonly ?LpaType $caseSubtype,
         public readonly ?string $channel,
         public readonly ?DateTimeImmutable $dispatchDate,
-        public readonly ?object $donor,
+        public readonly ?person $donor,
```

Comment:
```suggestion
        public readonly ?Person $donor,
```

PHP is case insensitive (urgh) so we need to make sure we pay special attention to that kind of thing.